api_app/mixins.py

```diff
@@ -681,3 +682,60 @@ def authentication_header(self) -> dict:
             return {"Auth-Key": self._service_api_key}
 
         return {}
+
+
+class JoeSandboxMixin:
+    _api_key: str
+    system_to_use: str = "lnxubuntu20"
+
+    def wait_for_analysis_to_finish(self, session: JoeSandbox, id: str) -> bool:
+        status: str = ""
+        while status != "finished":
+            try:
+                status = session.submission_info(submission_id=id)["status"]
+                logger.info("Polling again after 60 seconds")
+                time.sleep(60)
```

Comment:
this value would be better as an optional parameter and configurable from the user if possible